atom/renderer/atom_sandboxed_renderer_client.cc

```diff
@@ -116,8 +116,10 @@ class AtomSandboxedRenderFrameObserver : public AtomRenderFrameObserver {
 
     auto* isolate = blink::MainThreadIsolate();
     v8::HandleScope handle_scope(isolate);
-    auto context = frame->MainWorldScriptContext();
+
+    auto context = renderer_client_->GetContext(frame, isolate);
```

Comment:
This might be creating ```Cannot get property 'TMPDIR' on missing remote object 59``` when trying to access object through electron.remote now.

Reply:
@dsabanin This change only landed 12 days ago and has only been released in 5.0.  The error you quote there wouldn't be caused by this PR afaik

Reply:
Thanks for taking a look!